### PR TITLE
RHTAPSRE-431: Migrating Konflux Public Production (ArgoCD) to appsrep05ue1

### PR DESCRIPTION
**Pull request by manish-jangra**
This PR will make the following changes:
- [x] This creates another overlay  for migrating public production argocd instances to appsre managed cluster
- [x] This will disable the auto sync option for production argocd

### CODE REVIEW

**Comment by gbenhaim** [comment]
@manish-jangra the changes to production and staging should be separated into two different prs.

**Comment by manish-jangra** [comment]
@gbenhaim opened another [pull request](https://github.com/redhat-appstudio/infra-deployments/pull/3450) to enable auto sync option for konflux public staging.

File: argo-cd-apps/overlays/konflux-public-production/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+resources:
+  - ../../base/host
+  - ../../base/member
+  - ../../base/all-clusters
+  - ../../base/tekton-ci
+  - ../../base/tenants-config
+  - ../../base/cluster-secret-store-rh
+  - ../../base/rh-managed-workspaces-config
+
+namespace: argocd
```
Comment (gbenhaim) [inline argo-cd-apps/overlays/konflux-public-production/kustomization.yaml:12]: is this the correct namespace?

Comment (gbenhaim) [reply argo-cd-apps/overlays/konflux-public-production/kustomization.yaml:12]: i see it's a duplicate. can you please remove it?

Comment (manish-jangra) [reply argo-cd-apps/overlays/konflux-public-production/kustomization.yaml]: accidentally added two namespace, fixed now.

**Pull request merged**